gui/qt/__init__.py

```diff
@@ -92,7 +92,8 @@ def __init__(self, config, daemon, plugins):
         #network.add_jobs([DebugMem([Abstract_Wallet, SPV, Synchronizer,
         #                            ElectrumWindow], interval=5)])
         QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_X11InitThreads)
-        QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_ShareOpenGLContexts)
+        if hasattr(QtCore.QCoreApplication, "AA_ShareOpenGLContexts"):
```

Comment:
it should be `if hasattr(QtCore.Qt, "AA_ShareOpenGLContexts"):`

Reply:
oh crap, so I did have it right the first time, but then I second guessed myself into breaking it =D

I'll rebase a99f237 on top of master and push --force that to this PR